sdk/tests/example.test.ts

```diff
@@ -0,0 +1,251 @@
+
+describe('sdk guide snippets', () => {
+  it('sdk guide', async () => {
+    const client = initCasperClient(LOCAL_NODE_URL)
+
+    const account = ALICE
+    const accountAddress = getAccountHashFromKey(account)
+
+    const receiver = BOB
+    const receiverAddress = getAccountHashFromKey(receiver)
+
+    const [INVARIANT_CONTRACT_PACKAGE, INVARIANT_CONTRACT_HASH] = await Invariant.deploy(
+      client,
+      Network.Local,
+      account,
+      0n,
+      600000000000n
+    )
+    const [TOKEN_0_CONTRACT_PACKAGE, TOKEN_0_CONTRACT_HASH] = await Erc20.deploy(
+      client,
+      Network.Local,
+      account,
+      'erc20-1',
+      1000000000000000n,
+      'Coin',
+      'COIN',
+      12n,
+      150000000000n
+    )
+    const [TOKEN_1_CONTRACT_PACKAGE, TOKEN_1_CONTRACT_HASH] = await Erc20.deploy(
+      client,
+      Network.Local,
+      account,
+      'erc20-2',
+      1000000000000000n,
+      'Coin',
+      'COIN',
+      12n,
+      150000000000n
+    )
+
+    // load invariant contract
+    const invariant = await Invariant.load(client, INVARIANT_CONTRACT_HASH, Network.Local)
+
+    // load token contract
+    const erc20 = await Erc20.load(client, Network.Local, TOKEN_0_CONTRACT_HASH)
+
+    // set fee tier, make sture that fee tier with specified parameters exists
+    const feeTier = await newFeeTier({ v: 1000000000n }, 1n) // fee: 0.01 = 1%, tick spacing: 1
+
+    // if the fee tier does not exist, you have to add it
+    const isAdded = await invariant.feeTierExist(feeTier)
+    if (!isAdded) {
+      await invariant.addFeeTier(account, feeTier)
+    }
+
+    // set initial price of the pool, we set it to 1.00, remember its square root of price
+    const initSqrtPrice = { v: 1000000000000000000000000n }
+
+    // set pool key, make sure that pool with speecified parameters does not exists
+    const poolKey = await newPoolKey(TOKEN_0_CONTRACT_PACKAGE, TOKEN_1_CONTRACT_PACKAGE, feeTier)
+
+    const createPoolResult = await invariant.createPool(account, poolKey, initSqrtPrice)
+
+    // print transaction result
+    console.log(createPoolResult.execution_results[0].result)
+
+    // token y has 12 decimals and we want to add 8 actual tokens to our position
+    const tokenYAmount: TokenAmount = { v: 8n * 10n ** 12n }
+
+    // set lower and upper tick indexes, we want to create position in range [-10, 10]
+    const lowerTickIndex = -10n
+    const upperTickIndex = 10n
+
+    // calculate amount of token x we need to give to create position
+    const { amount: tokenXAmount, l: positionLiquidity } = await getLiquidityByY(
+      tokenYAmount,
+      lowerTickIndex,
+      upperTickIndex,
+      initSqrtPrice,
+      true
+    )
+
+    // print amount of token x and y we need to give to create position based on parameters we passed
+    console.log(tokenXAmount, tokenYAmount)
+
+    const [TOKEN_X_CONTRACT_HASH, TOKEN_Y_CONTRACT_HASH] = (await isTokenX(
```

Comment:
This is often repeated, it is worth adding a util function that does the order:
`orderTokens(TOKEN_0_CONTRACT_PACKAGE, TOKEN_1_CONTRACT_PACKAGE] -> [TOKEN_X_CONTRACT_HASH, TOKEN_Y_CONTRACT_HASH]`